pkg/app/app.go

```diff
@@ -89,6 +90,10 @@ func DefineStartCommandFlags(kpApp *kingpin.Application, cmd *kingpin.CmdClause)
 		Default(Helm3Timeout.String()).
 		DurationVar(&Helm3Timeout)
 
+	cmd.Flag("helm-ignore-release", "Helm3+: do not count release as a module release.").
```

Comment:
This change should be mentioned in the description of the PR.

Reply:
Done